src/tojson.jl

```diff
@@ -100,10 +100,22 @@ function tojson(output::String, ts::ReportingTestSet)
                 status = "fail"
                 message = string(result)
                 test_code = "@test " * result.orig_expr
+            elseif result isa Test.LogTestFailure
+                status = "fail"
+                message = string(result)
+                test_code = "@test_log " * string(result.orig_expr)
```

Comment:
```suggestion
                test_code = "@test_logs $(join(result.patterns, ' ')) $(result.orig_expr)"
```

Untested, looks right?

Doesn't really matter because we'll probably not use `@test_logs` anyway.